app/src/hid_listener.c

```diff
@@ -66,14 +66,25 @@ static int hid_listener_keycode_released(const struct zmk_keycode_state_changed
         return err;
     }
 
+#if IS_ENABLED(CONFIG_ZMK_HID_SEPARATE_MOD_RELEASE_REPORT)
+
+    // send report of normal key release early to fix the issue
+    // of some programs recognizing the implicit_mod release before the actual key release
+    err = zmk_endpoints_send_report(ev->usage_page);
+    if (err < 0) {
+        LOG_ERR("Failed to send key report for release keycode for consumer page event (%d)", err);
```

Comment:
This log message seems incorrect.

Reply:
Wow, that was quick.
I copy pasted that from below and changed 'changed modifiers' to released key as I wasn't really sure what else to log there.

Reply:
You know the code change better than I do :) but the "consumer page event" part seems incorrect, at least.

Reply:
"Failed to send separate mod release report (%d)" might be good enough.

Reply:
or what about 'Failed to send key report for the released key (%d)'
because it's sending the released key first with the new call, and in the return it sends the modifier changes.